lib/cocoapods/installer.rb

```diff
@@ -279,8 +279,18 @@ def validate_build_configurations
     #
     def clean_sandbox
```

Comment:
this method was updated because `sandbox#prepare` no longer removes the entire folder. But it will clean up target support files for all pods or aggregate that have been removed to ensure we do not leave unneeded files to the users `Pods/` folder.